src/logic/operate.js

```diff
@@ -0,0 +1,30 @@
+import big from 'big.js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  const numOneEval = big(numberOne);
+  const numTwoEval = big(numberTwo);
+  let finalOpr;
+  switch (operation) {
+    case '+':
+      finalOpr = numOneEval.plus(numTwoEval).toString();
+      break;
+    case '-':
+      finalOpr = numOneEval.minus(numTwoEval).toString();
+      break;
+    case 'x':
+      finalOpr = numOneEval.times(numTwoEval).toString();
+      break;
+    case '÷':
+      finalOpr = numOneEval.div(numTwoEval).toString();
+      break;
```

Comment:
`Warning :` ==> dont forget to manage division by ZERO on milestone 5 else, you'll get an error

Reply:
Thank you for the warning. 👍 
I've taken care of that.